docs/pebble.rst

```diff
@@ -0,0 +1,6 @@
+.. _ops_pebble_module:
+
+ops.pebble
```

Comment:
This shows up as "ops.pebble" in the sidebar, which doesn't fit the others. Can we make it "Pebble client"?